### PR TITLE
Issue #228

### LINKED ISSUE #225: Change place holder Images Nodes
No nodes, desing: [https://zpl.io/anqApnO](https://zpl.io/anqApnO)
Deleted Node, design: [https://zpl.io/bAo0BNe](https://zpl.io/bAo0BNe)

### LINKED ISSUE #228: UI fix Nodes/Choose Chains
Page: https://dashboard.pokt.network/dashboard/nodes/new/chains

1. Align the checkboxes column to be aligned with the search bar and extend the search bar through the whole white container. see design: https://zpl.io/VxzgLpx
2. SERVICE URL
Please add as place holder https://example.com:443
What is service URL, link to https://docs.pokt.network/docs/faq-for-nodes#section-what-is-the-service-uri
Add the warning box as it is on the design, Click here>> Link: https://docs.pokt.network/docs/testing-your-node

### PR DESCRIPTION
**Pull request by wgarcia4190**
Closes #228

### CODE REVIEW

**Review by pabelnl — DISMISSED** [review]

File: frontend/src/views/Nodes/CreateNodeForm/CreateNodeForm.js [hunk 1]
```diff
@@ -173,139 +176,112 @@ class CreateNodeForm extends CreateForm {
...
+                              isInvalid={!!errors.contactEmail}
+                            />
+                            <Form.Control.Feedback type="invalid">
+                              {errors.contactEmail}
+                            </Form.Control.Feedback>
+                          </Form.Group>
+                        </Col>
+                      </Row>
+                      <Form.Group>
+                        <Form.Label>Write an optional descriptio of your app here</Form.Label>
```
Comment (nymd) [inline frontend/src/views/Nodes/CreateNodeForm/CreateNodeForm.js]: Missing n on description

File: frontend/src/views/Nodes/NodeChainList/NodeChainList.js [hunk 2]
```diff
@@ -79,10 +80,7 @@ class NodeChainList extends Chains {
           <Col className="page-title">
             <h1>Choose chains</h1>
             <p>
-              Choose the chains you want to connect your node to. Remember you
-              won&#39;t be able to change these chains unless you unstake then
-              restake which will be evenly divided across the selected number of
-              chains.
+              Choose the chains you want to connect your app or node to. Remember you won&#39;t be able to change these chains until your next stake which will be evenly divided on the selected number of chains.
             </p>
           </Col>
```
Comment (nymd) [inline frontend/src/views/Nodes/NodeChainList/NodeChainList.js:85]: Choose the chains that your Pocket Node will service. Your staked POKT will be evenly divided between these selections. You will not be able to change this selection unless you unstake and restake your node.

File: frontend/src/views/Nodes/NodeChainList/NodeChainList.js [hunk 3]
```diff
@@ -145,12 +148,11 @@ class NodeChainList extends Chains {
                 <Form noValidate onSubmit={handleSubmit}>
                   <Form.Group>
                     <Form.Label className="service-url-label">
-                      Service URL. Please provide the HTTPS endpoint of your
-                      node.
+                      Please provide the endpoint HTTPS of your node. <a href="https://docs.pokt.network/docs/faq-for-nodes#section-what-is-the-service-uri">What is the service URL?</a>
```
Comment (nymd) [inline frontend/src/views/Nodes/NodeChainList/NodeChainList.js]: Please provide the HTTPS endpoint of your Pocket Node.

File: frontend/src/views/Nodes/NodeChainList/NodeChainList.js [hunk 4]
```diff
@@ -159,6 +161,31 @@ class NodeChainList extends Chains {
...
+
+                  <Row className="mt-4">
+                    <Col style={{paddingLeft: "0px"}}>
+                      <AppAlert
+                        className="pb-4 pt-4"
+                        variant="warning"
+                        title={
+                          <>
+                            <h4 className="text-uppercase">
+                              WARNING, BEFORE YOU CONTINUE!!{" "}
```
Comment (nymd) [inline frontend/src/views/Nodes/NodeChainList/NodeChainList.js]: Only one! Please

File: frontend/src/views/Nodes/NodeChainList/NodeChainList.js [hunk 5]
```diff
@@ -159,6 +161,31 @@ class NodeChainList extends Chains {
...
+                            <p className="ml-2">
+                            </p>
+                          </>
+                        }
+                      >
+                        <p>
+                          The key file by itself is useless without the passphrase.
+                          You&#39;ll need the key file in order to import or set up
+                          your node.
+                Before continue, be aware that we are not responsible of any jailing or slashing that may incur due to mis-configuration of your node. If you are not completely sure if your node is configured, <a href="https://docs.pokt.network/docs/testing-your-node">click here</a> and make sure you have done all of the steps and tested your node BEFORE you continue.
```
Comment (nymd) [inline frontend/src/views/Nodes/NodeChainList/NodeChainList.js]: Before continuing...

**Review by nymd — CHANGES_REQUESTED** [review]

**Review by pabelnl — APPROVED** [review]

**Review by nymd — APPROVED** [review]

**Pull request merged**